### DISCUSSION TITLE
Individual segmentation number of PT FDG PET Tumor model output segmentation

### DISCUSSION THREAD

**Discussion by wurunze** [ENHANCE-PET/MOOSE#85] [General]
I used pt_fdg_tumor model to segment PET images and read the resulting mask file with 3D Slicer. The segmentation looks satisfying, but all lesions have a same mask number, i.e. all lesions were included into one mask. It will be very helpful to generate masks for each lesion (one lesion's mask has unique segmentation mask index). Such that we can get measurement for each lesion using a software liking 3D Slicer. Is it possible with moosez?

**Comment by LalithShiyam** [comment]
Hi @wurunze many thanks for reaching out. Tumor segmentation is already discontinued from our moose tool from the latest version. We are creating a new tool for tumor segmentation with various tracers (FDG, PSMA... etc.,) in mind. 

Regarding your question, What you ask for is an instance segmentation, which is not planned for. If it's urgent I would use a 3d connected components analysis to quickly label them individually. Check this out: https://discourse.slicer.org/t/how-do-we-extract-the-features-for-each-independent-object-in-an-image/12120
Hope this helps.
Cheers,
Lalith
